examples/installation/multi_account_deployment/variables.tf

```diff
@@ -158,4 +158,14 @@ variable "hub_skip_instance_health_verification" {
 variable "gw_skip_instance_health_verification" {
   default     = false
   description = "This variable allows the user to skip the verification step that checks the health of the gw instance after it is launched. Set this variable to true to skip the verification, or false to perform the verification. By default, the verification is performed. Skipping is not recommended"
+}
```

Comment:
I don't think we should put these variables in the examples we provide. I think the customer should add this var in case he must use it. This is an option for experts. It will confuse the noobs.